File: 1st Semester/Computer Fundamentals I/Unit 1/C/For/Sumatoria_Fracciones.c
/*
 Sumatoria de 1/1 + 1/2 + 1/3 + 1/4 + ... + 1/50
 Calcular el resultado
*/
#include <stdio.h>
int main () {
    double i = 1, j = 1, sumatoria = 0;
    printf("%d/%d", j, i);
    for (i = 1; i < 51; i++) {
        printf(" + %.0f/%.0f", j, i);
        sumatoria += j / i;
    }
    printf("\nSumatoria: %f", sumatoria);
    return 0;
}

File: 1st Semester/Computer Fundamentals I/Unit 1/Java/For/Sumatoria_Fracciones.java
/*
 Sumatoria de 1/1 + 1/2 + 1/3 + 1/4 + ... + 1/50
 Calcular el resultado
*/
public class Sumatoria_Fracciones {
    public static void main (String[] args) {
        double n = 1, i = 1, resultado = 0;
        System.out.println("\n|| SUMATORIA DE FRACCIONES ||\n");
        System.out.printf("%.0f", i / n);
        for (n = 1; n < 50; ) {
            n++;
            System.out.printf(", %.3f", i / n);
            resultado += i / n;
        }
        System.out.printf("\nResultado de la sumatoria: %.3f", resultado);
    }
}